cloudinit/sources/DataSourceAzure.py

```diff
@@ -633,6 +670,10 @@ def crawl_metadata(self):
 
         imds_md = {}
         if self._is_ephemeral_networking_up():
+            # check if azure-proxy-agent is enabled in the ovf-env.xml file.
```

Comment:
Let's comment that this is disabled by default, and customer opt-in please.

Reply:
done